sql3/test/defs/defs_timequantum.go

```diff
@@ -111,6 +154,20 @@ var timeQuantumTest = TableTest{
 			),
 			Compare: CompareExactUnordered,
 		},
+		{
```

Comment:
This comment is for the StringSetQ query test. Based on my understanding of where `rangeq(a.ss1, '2022-01-02T00:00:00Z', null)` the result should include `'test4'` and `'test5'`, because the timestamp on those records are on or after (>=) the `from` parameter value `'2022-01-02'`. I may be misunderstanding how rangeq() works.